src/main/java/rx/schedulers/Schedulers.java

```diff
@@ -32,7 +35,14 @@
     private final Scheduler ioScheduler;
     private final Scheduler newThreadScheduler;
 
-    private static final Schedulers INSTANCE = new Schedulers();
+    private static Schedulers INSTANCE;
+
+    private static synchronized Schedulers getInstance() {
+        if (INSTANCE == null) {
+            INSTANCE = new Schedulers();
+        }
+        return INSTANCE;
+    }
```

Comment:
Two problems: 

  - `shutdown()` is public API and that would be an incompatible change
  - you'd have to expose `Schedulers` as an instance somehow call an instance method on it.